acceptance/setup/pre_suite/40_install_deps.rb

```diff
@@ -47,7 +47,13 @@
       if test_config[:install_type] == :git then
         case os
         when :debian
-          on database, "apt-get install -y --force-yes openjdk-7-jre-headless rake unzip"
+          if database.['platform'].variant == 'debian' &&
```

Comment:
Looks like Jenkins CI on the PR is failing because of this line:

```
SyntaxError: /tmp/jenkins/workspace/enterprise_puppetdb_integration-system-puppetdb_full-master/HYPERVISOR/abs/INSTALL_TYPE/install/LAYOUT/centos7-64mda-64a/NIGHTLY/false/PUPPETDB_DATABASE/postgres/label/beaker/acceptance/setup/pre_suite/40_install_deps.rb:50: syntax error, unexpected '[', expecting '('
          if database.['platform'].variant == 'debian' &&
```

Maybe just need to change this to `if database['platform'].variant ...` -- removing the "." after "database"?